### ISSUE TITLE
Does BatchNormalization in DL4J has bias or weights as parameters other than running mean or running var ?

### ISSUE THREAD

**Issue by nitin2212** [deeplearning4j/deeplearning4j#8980]
I wanted to see how BatchNorm implementation is different from other frameworks like PyTorch. PyTorch uses weight and bias by default while DL4J does not. I observed this using ResNet50 model param table. 

Please help !

**Comment by agibsonccc** [comment]
@nitin2212  please post questions like this on the dl4j forum. Was the question how dl4j implements batch norm? I think you answered it yourself. I'm closing this.
